runtime/doc/eval.txt

```diff
@@ -7052,6 +7073,26 @@ sqrt({expr})						*sqrt()*
 		"nan" may be different, it depends on system libraries.
 
 
+stdioopen({opts})			 *stdioopen()*
+		In a nvim launched with the |--headless| option, this opens
+		stdin and stdout as a |channel|. This function can only be
+		invoked once per instance. See |channels-stdio| for more
+		information and examples. See also |v:stderrchan|
+
+		Returns a |channel| ID. Close the stdio descriptors with |chanclose()|.
+		Use |chansend()| to send data stdout, and
+		|rpcrequest()| and |rpcnotify()| to communicate with a RPC
+		soceket.
+
+		{opts} is a dictionary with these keys:
+		  |on_stdio|: callback invoked when stdin is written to.
```

Comment:
`sockconnect()` uses `on_data`, should this be used here too? Otherwise `on_stdin`?